--- cmake/AwsCFlags.cmake	2021-01-10 22:22:11.000000000 +0100
+++ cmake/AwsCFlags.cmake	2021-01-10 22:22:38.000000000 +0100
@@ -53,7 +53,7 @@
         list(APPEND AWS_C_FLAGS "${_FLAGS}")
 
     else()
-        list(APPEND AWS_C_FLAGS -Wall -Werror -Wstrict-prototypes -fno-omit-frame-pointer)
+        list(APPEND AWS_C_FLAGS -Wall -Wstrict-prototypes -fno-omit-frame-pointer)
 
         if(NOT SET_PROPERTIES_NO_WEXTRA)
             list(APPEND AWS_C_FLAGS -Wextra)
