--- cmake/AwsCFlags.cmake	2020-11-16 12:13:54.000000000 +0100
+++ cmake/AwsCFlags.cmake	2020-11-16 12:14:21.000000000 +0100
@@ -48,7 +48,7 @@
         list(APPEND AWS_C_FLAGS "${_FLAGS}")
 
     else()
-        list(APPEND AWS_C_FLAGS -Wall -Werror -Wstrict-prototypes)
+        list(APPEND AWS_C_FLAGS -Wall -Wstrict-prototypes)
 
         if(NOT SET_PROPERTIES_NO_WEXTRA)
             list(APPEND AWS_C_FLAGS -Wextra)
@@ -96,7 +96,7 @@
             # some platforms implement htonl family of functions via GNU statement expressions (https://gcc.gnu.org/onlinedocs/gcc/Statement-Exprs.html)
             # which generates -Wgnu-statement-expression warning.
             set(old_flags "${CMAKE_REQUIRED_FLAGS}")
-            set(CMAKE_REQUIRED_FLAGS "-Wgnu -Werror")
+            set(CMAKE_REQUIRED_FLAGS "-Wgnu")
             check_c_source_compiles("
             #include <netinet/in.h>
 
